packages/lesswrong/components/localGroups/CommunityMap.tsx

```diff
@@ -151,7 +153,7 @@ const CommunityMap = ({ groupTerms, eventTerms, initialOpenWindows = [], center
         {...viewport}
         width="100%"
         height="100%"
-        mapStyle={"mapbox://styles/habryka/cilory317001r9mkmkcnvp2ra"}
+        mapStyle={isEAForum ? undefined : "mapbox://styles/habryka/cilory317001r9mkmkcnvp2ra"}
```

Comment:
You could plausibly try to MUI-fy the icons a bit, to make it match the rest of our icons.